core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -3232,6 +3232,24 @@ private[spark] object Utils extends Logging {
       case _ => math.max(sortedSize(len / 2), 1)
     }
   }
+
+  /**
+   * Return whether we are using G1GC or not
+   */
+  val isG1GC: Boolean = {
```

Comment:
This is OK here, though if it's only used here, maybe leave it in MemoryManager until it has reason to be shared? Utils is quite big already